App updates and app additions (#241)
riverar/wsa-app-compatibility: README.md

| 23andMe | 5.114.0 | 11 | ✅ |
| 4PDA | 1.9.35 | 11 | ✅ |
| A+ Gallery | 2.2.55.4 | 11 | ✅ | You might face graphical glitches when using dark theme, hence its recommended to use light theme instead.

> | Activity Launcher | 1.14.6 | 12 | ⚠️ | As of WSA version 2209.40000.28.0, you can't pin shortcuts to a home area of a launcher (like lawnchair) with the error `Current launcher does not support PinShortcut. Unable to create shortcut.` | You can still launch specific activities on any app 

Reviewer: ```suggestion
| Activity Launcher | 1.14.6 | 12 | ⚠️ | As of WSA version 2209.40000.28.0, you can't pin shortcuts to a home area of a launcher (like Lawnchair) with the error `Current launcher does not support PinShortcut. Unable to create shortcut.` | You can still launch specific activities on any app 
```

| BBC iPlayer | 4.137.0.25403 | 11 | ✅ | Sideloaded
| Berry Browser | 3.57.8 | 11, 12 | ✅
| Binance | 2.36.5 | 11 | ✅

> | Bondee | 2.0.0.636 | 12 | ⚠️ | App starts (with the intro), after the intro (the part where you sign in) it uses huge elements (like icons and text rather than the default setting) that is near impossible to navigate (Tried resizing the window size to no avail).

Reviewer: ```suggestion
| Bondee | 2.0.0.636 | 12 | ⚠️ | Has scaling issues. Specifically, after the introductory part, the app uses abnormally huge elements which makes it impossible to navigate.
```

| huaCtrl PRO | 1.0.27 | 11 | ✅
| Huawei AppGallery | 11.4.2.300 | 11 | ✅ | Frequent crashes were experienced, otherwise the app functionality is fine
| Hyper Square | 3.0.1 | 11 | ✅

> | IFTTT | 4.29.2 | 12 | 🆖 | Need GMS to receive notification. Ignore the Notification Reader Access error. | To avoid Play Protect blocking login to the Google Store, use GMS version open_gapps-x86_64-11.0-pico-20220215. (See also: WSAGAScript issue 213). 

Reviewer: ```suggestion
| IFTTT | 4.29.2 | 12 | 🆖 | Need GMS to receive notification. Ignore the Notification Reader Access error. | To avoid Play Protect blocking login to the Google Store, use GMS version open_gapps-x86_64-11.0-pico-20220215. (See also: WSAGAScript issue #213). 
```

| Trust: Crypto & Bitcoin Wallet | 6.57.1 | 12 | ✅ || for login, you have to go to Android settings => System => Date & Time and toggle the "Set Time Automatically" option. you can access it by this comman .\adb.exe shell "am start -n com.android.settings/.Settings"
| Tune In Pro | 28.7 (267721) | 11 | ✅
| Twitter | 9.16.1-release.00 | 11 | ✅ | Optionally requires GMS

> | Twitter Lite | 3.1.1 | 12 | ✅ || Starts as a PWA (Progressive Web App) like Kiwi Browser instead of a real app.

Reviewer: Twitter Lite is a PWA, right? I don't see what's wrong here. Mind elaborating?

> | Twitter Lite | 3.1.1 | 12 | ✅ || Starts as a PWA (Progressive Web App) like Kiwi Browser instead of a real app.

Reviewer: ```suggestion
| Twitter Lite | 3.1.1 | 12 | ✅ ||
```
